youtube_dl/extractor/go90.py

```diff
@@ -70,6 +76,16 @@ def _real_extract(self, url):
                             'height': int_or_none(source.get('height')),
                             'tbr': int_or_none(source.get('bitrate')),
                         })
+
+                # parse subtitles
+                for caption in asset.get('caption_metadata', []):
```

Comment:
as the `ext` is already present in the subtitle url it might be better to let youtube-dl extract it directly from the url(in case that start serving other types of subtitles) instead of the hardcoded value.